extras/cronwrapper

```diff
@@ -51,11 +51,11 @@ if $LOGGING; then
     RET=1
   fi
 else
-  "${SCRIPT}" "${CONF}" 2>&1 >${LOGFILE}
+  "${SCRIPT}" "${CONF}" >${LOGFILE} 2>&1
   RET=$?
 fi
 
-if [ $RET -ne 0 -a $RET -ne 2 -a $RET -ne 5 ]; then
+if [ $RET -ne 2 -a $RET -ne 5 ]; then
```

Comment:
But that means success will cause email, which seems odd. I only want emails if things go badly.